docs/install/install_everest_helm_charts.md

```diff
@@ -89,23 +101,54 @@ Here are the steps to install Percona Everest and deploy additional database nam
                 everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:8080/TCP    10s
                 ```
 
+        3. Run the following command to connect to Percona Everest when **TLS is enabled**:       
+                    
+            ```sh
+            kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "LoadBalancer"}}'
+            ```
+
+            ??? example "Expected output"
+                ```sh 
+                kubectl get svc/everest -n everest-system
+                ```
+                ```
+                NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+                everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:443/TCP    10s
+                ```
+
+            This is the address where you can then launch Everest at the end of the installation procedure. In this example, the external IP address used is [http://34.175.201.246](http://34.175.201.246).
+
+            For comprehensive instructions on enabling TLS for Percona Everest, see the section [TLS setup with Percona Everest](../security/tls_setup.md#tls-setup-with-percona-everest).
```

Comment:
The `patch` command does not change regardless of whether TLS is enabled or not, only the ports change. So no need to have a separate `step 3`, it can be merged into the same `Expected output`

Reply:
Done